src/components/draggable-overlay/index.tsx

```
@@ -134,7 +134,10 @@ const actions: ActionMap<State, Actions> = {
 interface DraggableOverlayProps {
   onClose?: () => void;
   heightPercentage?: number;
-  children: ((handleClose: () => void) => React.ReactNode) | React.ReactNode;
+  children:
+    | ((handleClose: (velocity?: number) => void) => React.ReactNode)
+    | React.ReactNode
+    | any;
```

why?

---

Whoops. Was testing around to suppress an error, thanks!